### PR TITLE
updated the readme file to update the installation instruction for a …

### PR DESCRIPTION
**Pull request by arunmastermind**
Updated Readme file to provide the instruction to install a specific version

### CODE REVIEW

**Comment by arunmastermind** [comment]
Updated Readme file to provide the instruction to install a specific version

**Comment by ImBIOS** [comment]
Thanks

**Comment by arunmastermind** [comment]
> Thanks

can we get this PR merged?

File: README.md [hunk 1]
```diff
@@ -26,3 +26,14 @@ bash <(curl -s https://raw.githubusercontent.com/pritunl/pritunl-client-electron
 ```bash
 bash <(curl -s https://raw.githubusercontent.com/pritunl/pritunl-client-electron/master/tools/uninstall_macos.sh)
 ```
+
+# Installing Specific Version
```
Comment (ImBIOS) [inline README.md]: Please ue heading 2 (`##`) here

**Review by ImBIOS — CHANGES_REQUESTED** [review]

**Comment by arunmastermind** [comment]
changes made as requested

**Comment by zachhuff386** [comment]
The script will already install the latest released version, it doesn't pull code directly from the repository. This is the only version I want recommended to install.

**Pull request closed without merging**